firefox-ios/Client/Frontend/Strings.swift

```diff
@@ -214,6 +214,11 @@ extension String {
                     tableName: "Bookmarks",
                     value: "Save sites as you browse. We’ll also grab bookmarks from other synced devices.",
                     comment: "The body text for the placeholder screen shown when there are no saved bookmarks, located at the root level of the bookmarks panel within the libray modal")
+                public static let ButtonTitle = MZLocalizedString(
+                    key: "Bookmarks.EmptyState.Root.ButtonTitle.v135",
+                    tableName: "Bookmarks",
+                    value: "Sign in to Sync",
+                    comment: "The button title for the sign in button on the placeholder screen shown when there are no saved bookmarks, located at the root level of the bookmarks panel within the libray modal. This button starts the initiates allowing users to sign in to their Mozilla Account to sync data")
```

Comment:
Typo + I am not sure I get the sentence "This button starts the initiates [...]"
```suggestion
                    comment: "The button title for the sign in button on the placeholder screen shown when there are no saved bookmarks, located at the root level of the bookmarks panel within the library modal. This button starts the flow allowing users to sign in to their Mozilla Account to sync data")
```

Reply:
woah, yea I don't understand what I was going for there either 😂